src/App.tsx

```diff
@@ -3,7 +3,7 @@ import { Global, jsx } from '@emotion/react'
 import React from 'react'
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'
 
-import LoadingSpinner from './components/loadingSpinner'
+import LoadingSpinner from './components/loading-spinner'
```

Comment:
Folders should be PascalCase: LoadingSpinner

filenames can be either PascalCase or camelCase
```suggestion
import LoadingSpinner from './components/LoadingSpinner'
```